### PR TITLE
[IMP] website: introduce `s_website_form_overlay` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new `s_website_form_overlay` snippet.

task-4138795
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/989

---
Capture d’écran 2024-10-14 à 15 15 57

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hey @odoo/rd-design 👋 
Could anyone check this new snippet before moving on to tech. review ? 
Thanks in advance !

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -327,6 +327,10 @@
                 </t>
                 <t t-snippet="website.s_opening_hours" string="Opening Hours" group="contact_and_forms"/>
                 <t t-snippet="website.s_contact_info" string="Contact Info" group="contact_and_forms"/>
+                <t t-snippet="website.s_website_form" string="Form" t-forbid-sanitize="form" t-thumbnail="/website/static/src/img/snippets_thumbs/s_website_form.svg" group="contact_and_forms"/>
```
Comment (mano-odoo) [inline addons/website/views/snippets/snippets.xml:330]: Check if necessary, it seems to be a leftover diff of the inner form snippet 😇

**Review by mano-odoo — CHANGES_REQUESTED** [review]
Just a single comment, the rest is perfect well done 💯

File: addons/website/views/snippets/snippets.xml (hunk 1 above)
Comment (Brieuc-brd) [reply addons/website/views/snippets/snippets.xml]: Good catch thanks ! I updated the branch 👍 
The diff: https://github.com/odoo/odoo/compare/2a48cbfb8299cfda3f690499fad208b43d1c2a87..9b198aab30cf52f26684762f80f61f30a0094ddd

**Review by mano-odoo — APPROVED** [review]
LGTM 😄

**Comment by Brieuc-brd** [comment]
Hey @odoo/rd-website 👋 
Here's a new snippet ready for tech. review, could you have a look at it ?
Thanks in advance !

**Comment by Brieuc-brd** [comment]
Added fallback image for industries + rebase

**Comment by lebl-odoo** [comment]
@Brieuc-brd 

I had a quick look and I'm wondering if we couldn't compute a distinct value for textarea to avoid a such layout distinction between those fields.
image

Moreover, is it possible to have an option to remove border radius on fields?

**Comment by mano-odoo** [comment]
> Moreover, is it possible to have an option to remove border radius on fields?

Hello @lebl-odoo, BRD is currently on holiday so I'm seizing the opportunity to answer 😄 

Currently you can tweak field border radius through the global input border radius option in the "Theme" tab.
image

But indeed it's not distinct at the moment, so you can't adjust solely the text-area. I agree that the text-area with a "pill" like border-radius looks off. So it could be interesting. 

I guess it's something we should check with @odoo/rd-website as well. 

cc @stefanorigano

**Comment by stefanorigano** [comment]
Yes, I agree with @mano-odoo’s analysis. However, the value @lebl-odoo set in his might be a bit too extreme (?).

I did a quick test with `20px`, and the text area appears rounded enough without looking off. While a dedicated controller for text areas could be useful (especially if the current setup doesn’t align with user expectations), I think we can achieve the desired “pill” shape with the existing options.


|`20px` |
|--------|
| Screenshot 2024-10-31 at 10 15 20 |

**Comment by lebl-odoo** [comment]
Oh, yeah sure :facepalm: Thx for the reminder.

> I guess it's something we should check with @odoo/rd-website as well.

I'll take the point on my own.

Thx

**Comment by Brieuc-brd** [comment]
Record issue on Cobalt theme fixed + branches rebased

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
Two small comments before I test and merge :pray:

**Comment by Brieuc-brd** [comment]
Thanks for the review @qsm-odoo 👍 
I've made the changes ([the diff](https://github.com/odoo/odoo/compare/f0c78373bd26237dd2ed66f25e6f0cecf8536848..62c6e855e1afb0e362e64573a6b3d4f616676dfe)) and rebased the branch ✅

**Review by qsm-odoo — APPROVED** [review]
Nice :+1: 

@robodoo r+

**Pull request closed without merging**